example/analysis

```diff
@@ -28,9 +28,10 @@ if (argLen === 3 &&
     usage()
 }
 
-// 60 seconds is a reasonable minimum timeout; For a "quick" analysis
-// the backend uses a value of 30 seconds.
-let timeout = 60
+// A "quick" analysis usually finishes within 90 seconds after the job starts.
+// A "full" analysis may run for 2 hours or more.
+
+let timeout = 86400
```

Comment:
Thanks for the updated information on what MythX execution times are. I was a bit unsure about that.  Is it really true the MythX can run for up to a day?

Reply:
I would say it shouldn't, realistically it would be up to ~3 hours in full mode unless it really get stuck (but then again, we don't really know :))

86400 is kind of synonymous for "run indefinitely, unless something really really isn't right".